taxi/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from taxi.views import index
+
+urlpatterns = [
+    path("", index, name='index'),
```

Comment:
Better not use different types of quotation marks in your code. Use double quotation marks everywhere. Check all files.

Reply:
Fixed